engines/bops_api/app/views/bops_api/v2/shared/_application.json.jbuilder

```diff
@@ -8,5 +8,7 @@ json.application do
   json.reference planning_application.reference
   json.fullReference planning_application.reference_in_full
   json.receivedAt planning_application.received_at
+  json.validFrom planning_application.validated_at
```

Comment:
Should we have this as `validAt` to keep consistent with other datetime fields we expose?

Reply:
Even `validFromAt` would be more consistent

Reply:
Hmm but the more I look at that the weirder it gets though.
Think `validAt` would suffice